docs/GettingStarted.md

```diff
@@ -257,11 +257,12 @@ This application loads the `cgroup_sock_addr.o` eBPF program and attaches to hoo
 eBPF for Windows uses ETW for tracing.  A trace can be captured in a file, or viewed in real-time.
 
 ### Capturing traces
-To capture a trace use the following commands:
-1) Start tracing: ```wpr.exe -start ebpfforwindows.wprp -filemode```
+To capture a trace in a file use the following commands:
+1) Start tracing: ```wpr.exe -start ebpfforwindows.wprp -filemode```. This will capture traces from eBPF execution context and the network eBPF extension drivers.
```

Comment:
```suggestion
1) Start tracing: ```wpr.exe -start ebpfforwindows.wprp -filemode```. This will capture traces from the core eBPF execution context and the network eBPF extension drivers.
```